wordpress-mobile/WordPress-Android
# Issue/6488 password reset link

### Fix
Added self-hosted site condition such that the lost password link directs to `[self-hosted site]wp-login.php?action=lostpassword` rather than `https://wordpress.com/wp-login.php?action=lostpassword` when applicable as described in
 https://github.com/wordpress-mobile/WordPress-Android/issues/6488.

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Tap ***Log in to your site by entering your site address instead.*** link.
3. Enter self-hosted site address.
4. Tap ***Next*** button.
5. Tap ***Lost your password?*** link.
6. Notice `[self-hosted site]wp-login.php?action=lostpassword` site displayed.
7. Tap back button twice.
8. Enter WordPress.com site address.
9. Tap ***Next*** button.
10. Tap ***Lost your password?*** link.
11. Notice `https://wordpress.com/wp-login.php?action=lostpassword` site displayed.
